core/types/avalanchego.go

```diff
@@ -0,0 +1,58 @@
+// (c) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package types
+
+import (
+	"crypto/ecdsa"
+	"math/big"
+
+	"github.com/ava-labs/libevm/common"
+	ethtypes "github.com/ava-labs/libevm/core/types"
+)
+
+// TODO(arr4n): this is a temporary workaround because of the circular
```

Comment:
Can't we make a change to avalanchego first to not import coreth types and import libevm instead, then have this PR without this file?

Reply:
I considered that but I'm pretty sure that `avalanchego` will only start using `libevm` transitively through `coreth` (currently its `go.mod` points to `ethereum/go-ethereum`). If we were to do it on a branch then the E2E test here would have to check out that branch instead, which would invalidate the test unless we kept said branch up to date (which we would almost certainly forget to do).

Reply:
Summarising our DM chat: I missed that here we depend on such a branch. I've opened ava-labs/avalanchego#3803 and will then remove all aliases here.

Reply:
Done.